runtime/vm/VMSnapshotImpl.cpp

```diff
@@ -374,6 +374,19 @@ VMSnapshotImpl::isImmortalClassLoader(J9ClassLoader *classLoader)
 	return isImmortal;
 }
 
+void
+VMSnapshotImpl::saveHiddenInstanceFields()
+{
+       _snapshotHeader->savedJavaVMStructs.hiddenInstanceFields = _vm->hiddenInstanceFields;
```

Comment:
@ThanHenderson Could you please create a follow-up change that uses tabs to indent here and line 386 (like the rest of this file)?

Reply:
Yes, I caught this too, post-merge, and have a patch for it. I'll open a PR shortly.